sdk/core/azure-core/inc/azure/core/internal/json/json.hpp

```diff
@@ -86,7 +86,8 @@
 
 #define _azure_NLOHMANN_JSON_ABI_TAGS \
   _azure_NLOHMANN_JSON_ABI_TAGS_CONCAT( \
-      _azure_NLOHMANN_JSON_ABI_TAG_DIAGNOSTICS, _azure_NLOHMANN_JSON_ABI_TAG_LEGACY_DISCARDED_VALUE_COMPARISON)
+      _azure_NLOHMANN_JSON_ABI_TAG_DIAGNOSTICS, \
+      _azure_NLOHMANN_JSON_ABI_TAG_LEGACY_DISCARDED_VALUE_COMPARISON)
```

Comment:
Why are we modifying this nlohmann json file in the Context PR?
If needed, should we do it in a separate PR to keep the focus of this PR on Context related changes?

Reply:
We're not - this must have snuck in due to a merge.

Reply:
Oh, I see what happened - clang-format kicked in and I didn't notice it.